docs/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,94 @@
+> Release Issue Template
+
+# go-ipfs X.Y.Z Release
+
+We're happy to announce go-ipfs X.Y.Z, bla bla...
+
+## 🗺 What's left for release
+
+<List of items with PRs and/or Issues to be considered for this release>
+
+## 🔦 Highlights
+
+< top highlights for this release notes >
+
+## Changelog
+
+< changelog generated by bin/mkreleaselog >
+
+## ✅ Release Checklist
+
+For each RC published in each stage:
+
+- version string in `version.go` has been updated
+- tag commit with vX.Y.Z-rcN
+- [ ] Stage 1 - Internal Testing
+  - [ ] Feature freeze. If any non-trivial features get added to the release, uncheck all the checkboxes and return to this stage.
```

Comment:
TODO: Add a footnote to explain what is a "non-trivial feature".

Reply:
Maybe "non-trivial non-bugfix changes"? I'm thinking about things like minor performance optimizations.